extensions/drupal-toolbox/src/components/EmptyView.tsx

```diff
@@ -25,8 +25,8 @@ function EmptyView(props: {
   return (
     <List.EmptyView
       icon="drupal-toolbox.png"
-      title="No Drupal websites found"
-      description="You don't have any Drupal websites yet. Why not add some?"
+      title="No Sites Found"
+      description="Press ⌘+N to add a site"
```

Comment:
Drupal, drupal, drupal, druuuupal: https://youtu.be/lZ-s3DRZJKY?t=11

Reply:
But okay, not a big deal. I'll stop being drupal partisan :)

Reply:
It's just to shorten the text for the user and this way makes the UI/UX a bit easier. Damn, that's a catchy song 😊

Reply:
Happy to your mind :)